scripts/e2e/run-e2e.sh

```diff
@@ -12,10 +12,12 @@ function DeleteOtherAGICVersions() {
 }
 
 function InstallAGIC() {
+    # Using 'applicationGatewayName' without providing 'subscription' and 'resource group' will make AGIC use values from azure.json
     [[ -z "${version}" ]] && (echo "version is not set"; exit 1)
-    [[ -z "${applicationGatewayId}" ]] && (echo "buiapplicationGatewayIdldid is not set"; exit 1)
+    [[ -z "${applicationGatewayName}" ]] && (echo "applicationGatewayName is not set"; exit 1)
     [[ -z "${identityResourceId}" ]] && (echo "identityResourceId is not set"; exit 1)
     [[ -z "${identityClientId}" ]] && (echo "identityClientId is not set"; exit 1)
+    [[ -z "${applicationGatewaySubnetPrefix}" ]] && (echo "applicationGatewaySubnetPrefix is not set"; exit 1)
```

Comment:
why it's mandatory for install(exit 1)?

Reply:
right now, this mandatory in helm when using appgw name instead of appgw id.